### ISSUE TITLE
Use correct lang attribute value for works

### ISSUE THREAD

**Issue by gestchild** [wellcomecollection/wellcomecollection.org#11627]
### What

Our web pages have a default language of English, specified on the html element with a [lang attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang) value of 'en'

In some places where the language changes (Work page headers and inside the viewer on items pages) we attempt to that change by wrapping the relevant content in its own lang attribute. This is important because it helps screen readers to use the correct pronunciation, but [an incorrect value is a WCAG failure](https://www.w3.org/WAI/standards-guidelines/act/rules/de46e4/).

Valid lang attribute values come from ISO 639-1 language codes standard.

We populate the lang attribute from a Work's language property, e.g. 
`"languages": [
{
"id": "fre",
"label": "French",
"type": "Language"
}
]`

However, this does not appear to use the same standard and so we can end up with incorrect values.

If possible we should update the work to use the ISO 639-1 standard. If not, we should map the possible values in a Work to the ISO 639-1 standard, so that we can populate the lang attribute with correct values.

### Where

[Catalogue work with French language](https://api.wellcomecollection.org/catalogue/v2/works/ezz3jwt8?include=identifiers%2Cimages%2Citems%2Csubjects%2Cgenres%2Ccontributors%2Cproduction%2Cnotes%2CformerFrequency%2Cdesignation%2Cparts%2CpartOf%2CprecededBy%2CsucceededBy%2Clanguages%2Choldings).

[Work page for the above item](https://wellcomecollection.org/works/ezz3jwt8).

The heading has a lang attribute value of 'fre'. The correct value for French in the ISO 639-1 standard is 'fr'.

<img width="720" alt="Image" src="https://github.com/user-attachments/assets/3809349e-d3e2-4725-b92c-1ad977ece4f0" />

**Comment by rcantin-w** [comment]
Find a MARC -> ISO 639-1 library?
